src/figuresClasses.ts

```diff
@@ -1,19 +1,62 @@
+function round(num: number): number {
+  return Math.floor(num * 100) / 100;
+}
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
 export class Triangle implements Figure {
-
+  shape: string;
+
+  constructor(public color: string, public a: number, public b: number,
+    public c: number) {
+    if (Math.min(a, b, c) <= 0
+    || Math.max(a, b, c) >= a + b + c - Math.max(a, b, c)) {
+      throw new Error('Incorrect sides');
```

Comment:
I suggest providing more detailed error descriptions.

From the checklist:
> 3. [CODE STYLE] - If you throw an error try to provide its detailed description in error message.

Reply:
yes, true. Be more descriptive